src/Entity/ElectionRound.php

```diff
@@ -7,8 +7,8 @@
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
- * @ORM\Entity
  * @ORM\Table(name="election_rounds")
+ * @ORM\Entity
```

Comment:
Why move this annotation under the `@ORM\Table` ?

Reply:
For consistency (i.e https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L30).